package.json

```diff
@@ -42,6 +42,7 @@
     "acceptancetest:release": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@release' --exit && ts-mocha packages/ws-server/tests/acceptance/index.spec.ts  -g '@release' --exit",
     "acceptancetest:release_http": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@release' --exit",
     "acceptancetest:release_ws": "nyc ts-mocha packages/ws-server/tests/acceptance/index.spec.ts  -g '@release' --exit",
+    "acceptancetest:release_light": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@mainnet-light' --exit",
```

Comment:
I don't this this command would pick up any acceptance tests as it has the `@mainnet-light` tag while acceptance tests have `@release-light`.

Reply:
Good catch.  Updated.